packages/flame/example/lib/main.dart

```diff
@@ -70,7 +70,7 @@ class MyGame extends BaseGame with DoubleTapDetector, TapDetector {
 
     final handled = components.any((c) {
       if (c is PositionComponent && c.toRect().overlaps(touchArea)) {
-        remove(c);
+        components.remove(c);
```

Comment:
again, another method that I decided not to proxy. open to suggestions

Reply:
Maybe we should start doing `c.remove()` in our examples, not sure which one is clearer.